### PR TITLE
Add additional env vars to author compose config.

### PR DESCRIPTION
**Pull request by samiwel**
### What is the context of this PR?
This PR makes changes to the compose config in order to enable the preview feature within Author.

### Dependencies
 - [x] https://github.com/ONSdigital/eq-author/pull/233
 - [x] https://github.com/ONSdigital/go-launch-a-survey/pull/31

### How to review 
All tests and checks should pass.

### CODE REVIEW

File: eq.yml [hunk 1]
```diff
@@ -72,11 +72,15 @@ services:
...
       REACT_APP_USE_MOCK_API: "false"
+      REACT_APP_PUBLISHER_URL: http://localhost:9000/publish
+      REACT_APP_GO_LAUNCH_A_SURVEY_URL: http://localhost:8000/quick-launch
     depends_on:
       - eq-author-api
+      - eq-survey-launcher
     restart: always
     networks:
       - eq-author
+      - eq-runner
```
Comment (jonnyshaw89) [inline eq.yml]: I don't think you need this as the author doesn't talk directly to runner, it goes via the client.

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: eq.yml (hunk 1 above)
Comment (samiwel) [reply eq.yml]: I wasn't sure about this. Is `go-launch-a-suvey` available to `eq-author` on `localhost:8000` even when it's not on the eq-runner network?

Comment (jonnyshaw89) [reply eq.yml]: it doesn't need to be available from author. as author is all running client side, its only the client that needs to access it?

Comment (samiwel) [reply eq.yml]: Hmm, so does that mean that `REACT_APP_GO_LAUNCH_A_SURVEY_URL` is wrong? localhost:8000 won't resolve to the correct URL from the client?

Comment (jonnyshaw89) [reply eq.yml]: No that is correct. the launcher is bound to port 8000

File: eq.yml [hunk 2]
```diff
@@ -72,8 +72,11 @@ services:
       REACT_APP_BASE_NAME: /eq-author
       REACT_APP_API_URL: http://localhost:4000/graphql
       REACT_APP_USE_MOCK_API: "false"
+      REACT_APP_PUBLISHER_URL: http://localhost:9000/publish
```
Comment (jonnyshaw89) [inline eq.yml]: this should be `http://eq-publisher:9000/publish` as the communication is within the docker network

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

**Review by jonnyshaw89 — APPROVED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**